rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+
+    @book.add_rental(self)
+    @person.add_rental(self)
```

Comment:
- [ ] int the constructor of the Rental class The rental should add itself to the @book and the @person achieving the 'belongs-to' association, please have a loot at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)